### ISSUE TITLE
Limited shoulder flexion

### ISSUE THREAD

**Issue by JillMiller** [obophenotype/human-phenotype-ontology#5905]
**Preferred term label:** Limited shoulder flexion

**Synonyms**

**Definition (free text, please give PubMed ID)** Reduced range of motion of the shoulder resulting in difficulty lifting a straightened arm in front of the body., PMID:32296967

**Parent term (use hpo.jax.org/app)** Limited shoulder movement HP:0006467

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**

**Your nano-attribution (ORCID)**

**Comment by pnrobinson** [comment]
added
